src/main/java/com/adyen/model/checkout/CreatePaymentLinkRequest.java

```diff
@@ -133,7 +130,7 @@ public void write(final JsonWriter jsonWriter, final RecurringProcessingModelEnu
 
             @Override
             public RecurringProcessingModelEnum read(final JsonReader jsonReader) throws IOException {
-                String value = jsonReader.nextString();
+                Object value = jsonReader.nextString();
```

Comment:
possible null pointer. why it is changed to Object from String?

Reply:
seems it was changed in the model generator, i will revert it back.
Regarding NPE for `JsonReader`, I don't think we need to handle it as it is called by GSON internally.